docs/en/getting-started/example-datasets/wikistat.md

```diff
@@ -64,3 +64,22 @@ clickhouse-local --query "
   WHERE length(values) = 5 FORMAT Native
 " | clickhouse-client --query "INSERT INTO wikistat FORMAT Native"
 ```
+
+Or loading the cleaning data:
+
+```
+INSERT INTO wikistat WITH
+    parseDateTimeBestEffort(extract(_file, '^pageviews-([\\d\\-]+)\\.gz$')) AS time,
+    splitByChar(' ', line) AS values,
+    splitByChar('.', values[1]) AS projects
+SELECT
+    time,
+    projects[1] AS project,
+    projects[2] AS subproject,
+    decodeURLComponent(values[2]) AS path,
+    CAST(values[3], 'UInt64') AS hits
```

Comment:
the number of columns doesn't match: in the original table we have 6 rows, but we insert 5 rows🙂

Reply:
@yariks5s, thanks for your reply.

In the [presentation](https://presentations.clickhouse.com/fosdem2023/#17), the SQL about creating table is as follows:

```sql
CREATE TABLE wikistat
(
    time DateTime CODEC(Delta, ZSTD(3)),
    project LowCardinality(String),
    subproject LowCardinality(String),
    path String CODEC(ZSTD(3)),
    hits UInt64 CODEC(ZSTD(3))
)
ENGINE = MergeTree
ORDER BY (path, time);
```

That's different from original table schema and the `size` field is missed.

Should we be consistency for the table schema in the presentation?

Reply:
I think that we need to make it the same as in the presentation